packages/slice-machine/src/modules/useSliceMachineActions.ts

```diff
@@ -1,31 +1,28 @@
 import { useDispatch } from "react-redux";
-import { LoadingKeysEnum } from "@src/modules/loading/types";
-import { ModalKeysEnum } from "@src/modules/modal/types";
-import { modalCloseCreator, modalOpenCreator } from "@src/modules/modal";
-import {
-  startLoadingActionCreator,
-  stopLoadingActionCreator,
-} from "@src/modules/loading";
+import { LoadingKeysEnum } from "./loading/types";
+import { ModalKeysEnum } from "./modal/types";
+import { modalCloseCreator, modalOpenCreator } from "./modal";
+import { startLoadingActionCreator, stopLoadingActionCreator } from "./loading";
 import {
   finishOnboardingCreator,
   sendAReviewCreator,
   skipReviewCreator,
   updatesViewedCreator,
   hasSeenTutorialsTooTipCreator,
-} from "@src/modules/userContext";
-import { refreshStateCreator } from "@src/modules/environment";
+} from "./userContext";
+import { refreshStateCreator } from "./environment";
 import {
   openSetupDrawerCreator,
   closeSetupDrawerCreator,
   toggleSetupDrawerStepCreator,
   checkSimulatorSetupCreator,
   connectToSimulatorIframeCreator,
-} from "@src/modules/simulator";
+} from "./simulator";
 import ServerState from "@models/server/ServerState";
-import { createCustomTypeCreator } from "@src/modules/availableCustomTypes";
```

Comment:
These import, I feel changing them makes total sense, if the imported stuff are on the same level or above I feel it's okay.

But I'm not sure to see the need to change them everywhere you see one :)